stripe.go

```diff
@@ -486,7 +486,25 @@ func (s *BackendImplementation) ResponseToError(res *http.Response, resBody []by
 	}
 	raw.E.Err = typedError
 
-	s.LeveledLogger.Errorf("Error encountered from Stripe: %v", raw.E.Error)
+	// The Stripe API makes a distinction between errors that were caused by
+	// invalid parameters or something else versus those that occurred
+	// *despite* valid parameters, the latter coming back with status 402.
+	//
+	// On a 402, log to info so as to not make an integration's log noisy with
+	// error messages that they don't have much control over.
+	//
+	// Note I use the constant 402 here instead of an `http.Status*` constant
+	// because technically 402 is "Payment required". The Stripe API doesn't
+	// comply to the letter of the specification and uses it in a broader
+	// sense.
+	if res.StatusCode == 402 {
+		s.LeveledLogger.Infof("User-compelled request error from Stripe: %v",
```

Comment:
I find the wording here a bit confusing. Maybe a standard format could help, along the lines of:

    “Stripe request GET /blah returned 402 User Error: Invalid Card Expiration”

With method, path, status, status text and error message being placeholders?